core/src/main/scala/org/apache/spark/ui/ConsoleProgressBar.scala

```diff
@@ -38,6 +38,8 @@ private[spark] class ConsoleProgressBar(sc: SparkContext) extends Logging {
   private val updatePeriodMSec = sc.conf.get(UI_CONSOLE_PROGRESS_UPDATE_INTERVAL)
   // Delay to show up a progress bar, in milliseconds
   private val firstDelayMSec = 500L
+  // Get the stderr (which is console for spark-shell) before installing RedirectConsolePlugin
+  private val console = System.err
```

Comment:
@yaooqinn so it does not affect shell console progress bar, I also pasted the manual test result in PR description.